### ISSUE TITLE
Add the ability to automatically pan Braille displays

### ISSUE THREAD

**Issue by slannon97** [nvaccess/nvda#11396]
### Is your feature request related to a problem? Please describe.
no
### Describe the solution you'd like
NVDA should provide the ability to automatically pan the Braille display after a certain interval of time specified by the user. The initial time period should be customizable in the Braille dialog, and commands on the Braille display could be used to start and/or stop the automatic panning mode. Other commands should exist that increase or decrease the automatic pan interval by 1 second. These could be customized in the Input Gestures dialog.
### Describe alternatives you've considered
The BrailleExtender add-on can add this feature, but this should be built directly into NVDA.

**Comment by Afik-souffir998** [comment]
It will be nice

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by josephsl** [comment]
CC @andre9642, any thoughts?

**Comment by zstanecic** [comment]
Hi People!

I recall that Braille extender has this feature.

It’s called auto scrolling.

It does exactly what Afik wants.

**Comment by Afik-souffir998** [comment]
Its have to be in NVDA itself

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Andre9642** [comment]
Indeed, this feature is available in Braille Extender since awhile. 
I think I could submit an implementation in NVDA Core without too much effort.
Are there other interested people who want this feature?
Personally I use this feature to read long texts (without speech).

**Comment by josephsl** [comment]
Hi, if so, I advise adding a command (if you haven’t done so) to start automatic braille panning (I can do something about adding BrailleNote specific gestures). Thanks.

**Comment by retrochiptuner** [comment]
Hello! I am an interested user. I have a Focus Blue 14. I have no problem in moving, but I have broken the displacement wheels of a PAC Mate 40. In this way the user should be able to configure every how many seconds advance, as well as increase and decrease.
